zipline/utils/events.py

```diff
@@ -301,8 +301,6 @@ class AfterOpen(StatelessRule):
     """
     A rule that triggers for some offset after the market opens.
     Example that triggers after 30 minutes of the market opening:
-
-    >>> AfterOpen(minutes=30)
```

Comment:
(If we do remove it, we should remove the sentence above referring to it, but it seems useful to have a usage example.)